plugins/modules/aci_endpoint_ip_tag.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Sabari Jaganathan <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_endpoint_ip_tag
+short_description: Manage Endpoint IP Tag (fv:EpIpTag)
+description:
+- Manage Endpoint IP Tag on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of the Tenant.
+    type: str
+  name:
+    description:
+    - The name of the Endpoint IP Tag.
+    type: str
+  endpoint_ip_address:
+    description:
+    - The IPv4 or IPv6 address of the Endpoint IP Tag.
+    type: str
+    aliases: [ ip_addr ]
+  vrf:
```

Comment:
is it required for vrf to exist? if so add section in notes

Reply:
Done